paddle/phi/kernels/sparse/unary_grad_kernel.h

```diff
@@ -77,5 +77,17 @@ void CastCsrGradKernel(const Context& dev_ctx,
                        DataType value_dtype,
                        SparseCsrTensor* dx);
 
+template <typename T, typename Context>
+void TransposeCooGradKernel(const Context& dev_ctx,
+                            const SparseCooTensor& dout,
+                            const std::vector<int>& perm,
+                            SparseCooTensor* dx);
+
+template <typename T, typename Context>
+void TransposeCsrGradKernel(const Context& dev_ctx,
+                            const SparseCsrTensor& dout,
+                            const std::vector<int>& perm,
+                            SparseCsrTensor* dx);
```

Comment:
we should have [rfc](https://github.com/PaddlePaddle/community/blob/master/rfcs/APIs/20220907_api_design_for_sparse_transpose.md) same with code, there are more parameter `const SparseCooTensor& x` and `const SparseCsrTensor& x` with respect to `TransposeCooGradKernel` and `TransposeCsrGradKernel`in rfc